openhtf/core/measurements.py

```diff
@@ -348,6 +348,12 @@ def __init__(self, description='', unit=units.NO_DIMENSION):
     self.description = description
     self.unit = unit
 
+  def __eq__(self, other):
```

Comment:
To handle Python 2, when implementing \_\_eq__, implement \_\_ne__ as well

```python
def __ne__(self, other):
  return not self == other
```

discussion: https://stackoverflow.com/questions/4352244/python-should-i-implement-ne-operator-based-on-eq

Reply:
(meant to “request changes)

Reply:
Done.